1.wave2_data/3.processes_features/README.md

```diff
@@ -4,7 +4,7 @@ In this module, we extract image feature data from the CellProfiler .sqlite file
 
 ## Pycytominer
 
-We use [Pycytominer](https://github.com/cytomining/pycytominer) to perform the merging and extraction of the Interstellar wave 2 single cell features.
+We use [Pycytominer](https://github.com/cytomining/pycytominer) to perform the annotation and output of the Interstellar wave 2 image features.
```

Comment:
Are single-cell features still used for this step?

Reply:
Nope, in wave 2 we are only extracting image features.